### PR TITLE
Updating tests with new fixes

### PR DESCRIPTION
**Pull request by ddutt**
We have to update the tests given the new speed computation logic, summarize values being floats due to new pandas version etc.

### CODE REVIEW

File: suzieq/poller/services/interfaces.py [hunk 1]
```diff
@@ -217,6 +217,8 @@ def _clean_cumulus_data(self, processed_data, raw_data):
 
             if entry['type'] == 'ether':
                 entry['type'] = 'ethernet'
+            if entry.get('type', '') == 'vxlan':
+                entry['speed'] = 0
```
Comment (claudious96) [inline suzieq/poller/services/interfaces.py:221]: I think it's better to use `NO_SPEED` or the function `_get_missing_speed_value()`
```suggestion
                entry['speed'] = NO_SPEED
```

File: suzieq/poller/services/interfaces.py [hunk 2]
```diff
@@ -399,8 +401,14 @@ def fix_junos_speed(entry):
                     vlan = 0
                     iftype = entry['type']
 
-                speed = lentry.get('logical-interface-bandwidth', [{}])[0] \
-                    .get('data', 0)
+                if iftype == 'subinterface':
+                    speed = lentry \
+                        .get('logical-interface-bandwidth', [{}])[0] \
+                        .get('data', entry.get('speed', -1))
```
Comment (claudious96) [inline suzieq/poller/services/interfaces.py:407]:
```suggestion
                        .get('data', entry.get('speed', MISSING_SPEED))
```

File: suzieq/poller/services/interfaces.py [hunk 3]
```diff
@@ -399,8 +401,14 @@ def fix_junos_speed(entry):
...
+                else:
+                    speed = lentry \
+                        .get('logical-interface-bandwidth', [{}])[0] \
+                        .get('data', 0)
```
Comment (claudious96) [inline suzieq/poller/services/interfaces.py:411]:
```suggestion
                        .get('data', NO_SPEED)
```

File: suzieq/poller/services/interfaces.py [hunk 4]
```diff
@@ -428,17 +436,18 @@ def fix_junos_speed(entry):
...
+                addrlist = []
                 for elem in afis:
                     afi_mtu = elem.get('mtu', [{}])[0].get(
                         'data', entry['mtu'])
                     afi = elem.get('address-family-name', [{}])[0] \
                         .get('data', '')
                     if afi == 'inet':
                         no_inet = False
-                        addrlist = elem.get('interface-address', [])
+                        addrlist += elem.get('interface-address', [])
```
Comment (claudious96) [inline suzieq/poller/services/interfaces.py:447]: Is `elem['interface-address']` always an array? If not it could crash here

**Review by claudious96 — APPROVED** [review]

File: suzieq/poller/services/interfaces.py (hunk 4 above)
Comment (ddutt) [reply suzieq/poller/services/interfaces.py:447]: yes, always an array. See the init above

Comment (claudious96) [reply suzieq/poller/services/interfaces.py:447]: That is `addrList`, I meant `elem['interface-address']` from the `processed_data`

**Review by claudiolor — APPROVED** [review]

**Review by LucaNicosia — APPROVED** [review]

**Pull request merged**